ansible_rulebook/schema/ruleset_schema.json

```
@@ -75,13 +75,33 @@
                         {"$ref": "#/$defs/none-action"},
                         {"$ref": "#/$defs/shutdown-action"}
                     ]
+                },
+                "once_within" : {
+                    "type": "string",
+                    "pattern": "^\\d+\\s(milliseconds|seconds|minutes|hours|days|millisecond|second|minute|hour|day)$"
```

`?` means the preceding character is optional. In this case I propose this more concise version: 
```suggestion
                    "pattern": "^\\d+\\s(milliseconds?|seconds?|minutes?|hours?|days?)$"
```